runtime/runtime/src/prefetch.rs

```diff
@@ -85,7 +89,10 @@ impl TriePrefetcher {
     /// Start prefetching data for processing the receipts.
     ///
     /// Returns an error if the prefetching queue is full.
-    pub(crate) fn input_receipts(&mut self, receipts: &[Receipt]) -> Result<(), ()> {
+    pub(crate) fn input_receipts(
```

Comment:
I think we should change the name of this function to something like `prefetch_data_for_receipts`.  But maybe not as part of this PR.  Up to you.